### ISSUE TITLE
Error when reporting someone.

### ISSUE THREAD

**Issue by JadssDev** [MrTigreroux/TigerReports#56]
When the error happens: When I try to report myself 2 times 1 after the other.

Error:
```
[05:25:36 INFO]: jadss_pt issued server command: /report jadss_pt bbb
[05:25:36 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'report' in plugin TigerReports v4.12.5
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:666) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1189) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1001) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [patched.jar:git-PaperSpigot-"4c7641d"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_275]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_275]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [patched.jar:git-PaperSpigot-"4c7641d"]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_275]
Caused by: java.lang.NoSuchMethodError: java.time.Duration.toSeconds()J
        at fr.mrtigreroux.tigerreports.utils.MessageUtils.getSecondsBetweenNowAndDate(MessageUtils.java:117) ~[?:?]
        at fr.mrtigreroux.tigerreports.objects.users.User.getCooldown(User.java:142) ~[?:?]
        at fr.mrtigreroux.tigerreports.commands.ReportCommand.onCommand(ReportCommand.java:50) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        ... 15 more
```

**Comment by JadssDev** [comment]
Notes: this is a bungeecord server, connected to the same MySQL, just installed the plugin

**Comment by MrTigreroux** [comment]
Is it a Spigot plugin connected to BungeeCord ?
Do you use Java 8 or higher on this server ?

**Comment by MrTigreroux** [comment]
Should be fixed in new [version 4.12.6](https://www.spigotmc.org/resources/tigerreports.25773/update?update=397212). Thank you for reporting it.
